app/views/layouts/_signin_signout.html.erb

```diff
@@ -29,16 +29,16 @@
 
 <% else %>
   <% if !isActivePage(root_path) %>
-    <a href="#header-login-form" data-toggle="modal" dialog-width="450px"><%= _('Sign in')%></a> /
-    <a href="#header-register-form" data-toggle="modal" dialog-width="450px"><%= _('Create account')%></a>
+    <a id="sign_in_modal" href="#header-login-form" data-toggle="modal" dialog-width="450px"><%= _('Sign in')%></a> /
+    <a id="create_account_modal" href="#header-register-form" data-toggle="modal" dialog-width="450px"><%= _('Create account')%></a>
   <% end %>
 <% end %>
 
 <% if !user_signed_in? && !isActivePage(root_path) then %>
-  <div id="header-login-form" class="modal" style="display:none">
+  <div id="header-login-form" class="modal" role="dialog" aria-labelledby="#sign_in_modal" style="display:none">
```

Comment:
These two should be `aria-labelledby="some_id`. I had added an `<h2>` to both of them that is only visible to screen readers: `<h2 aria-hidden="false" class="aria-only"><%= _('Sign in') %></h2>`
So just add an id to those I think and use the aria-labelledby attribute. Then make sure the focus is set on the first element in the forms (I think it should be by default but best to verify)